module-3/back-end/index.js

```diff
@@ -85,6 +87,60 @@ app.put('/updateDocumentData/:id', async (req, res) => {
   }
 })
 
+app.post('/createProject', async (req, res) => {
+  try {
+    project = {
+      title: req.body.title,
+      description: req.body.description
+    }
+    const result = await Project.create(project)
+    res.status(200).json({result: result})
```

Comment:
1. When returning an object where key and variable name are the same, use a short syntax `.json({result})`
2. I don't think we want to wrap the result in an extra level of nesting here, I think returning it as `.json(result)` would be a little cleaner approach